versioned_docs/version-2.0.0/running-keploy/configuration-file.md

```diff
@@ -17,19 +17,79 @@ Introducing **Keploy-config** 🎉 - It is a YAML-based file that will allow you
 
 ## Getting Started:
 
+We will be using a sample app to demonstrate working of Keploy configuration file.
+
 To generate a keploy-config file, run:
 
 ```bash
-keploy generate-config --path "/path/to/your/project"
+keploy generate-config --path "."
 ```
 
+For demonstration purposes, we are using the root directory of the application. We can place it wherever we want to inside the project
```

Comment:
Since you are using echo-sql sample app! Please mention the same here with the url to the sample-app